### ISSUE TITLE
Update mind-ar-ts to newer version

### ISSUE THREAD

**Issue by demisquare** [krsbx/mind-ar-ts#119]
Hi, are there some plans to update mind-ar-ts to last mind-ar-js version?

**Comment by krsbx** [comment]
@demisquare yup, I'll try to update it in this repo ASAP, but if you want to contribute it also fine
